files/packit-httpd.conf

```diff
@@ -25,6 +25,21 @@ LogLevel wsgi:debug
 ErrorLog /dev/stderr
 TransferLog /dev/stdout
 
+# These directives are here to lower the memory usage.
+# For their description see https://httpd.apache.org/docs/2.4/mod/mpm_common.html
+# Note, Fedora's httpd is compiled with event MPM
+# https://httpd.apache.org/docs/2.4/mod/event.html
+# Number of child server PROCESSES created at startup
+StartServers 1
```

Comment:
From the docs:

> The StartServers directive sets the number of child server processes created on startup. As the number of processes is dynamically controlled depending on the load, (see MinSpareThreads, MaxSpareThreads, MinSpareServers, MaxSpareServers) **there is usually little reason to adjust this parameter**.

Reply:
Yes, **usually**, if you have normal/big load it makes no sense to set the initial value, because httpd will increase it later, **but** if we have a minimal load, setting it to minimal value (`1` in our case) IMHO makes `httpd` to not start more than necessary if you'll never need it (and `httpd` probably won't increase it due to our minimal load).